frontend/src/queries/Query/Query.tsx

```diff
@@ -98,7 +98,7 @@ export function Query<Q extends Node>(props: QueryProps<Q>): JSX.Element | null
         component = (
             <DataTable
                 query={query}
-                setQuery={setQuery as ((query: DataTableNode) => void) | undefined}
+                setQuery={setQuery as unknown as (query: DataTableNode) => void}
```

Comment:
also why do we need to cast to `unknown` now as we've just prevented `setQuery` from being `undefined` 😅

Reply:
Yeah, no idea, but the editor didn't like the direct cast for some reason. Couldn't figure out why removing `undefined` would change it so much 🤯